src/Clerk.ts

```diff
@@ -204,36 +218,45 @@ export default class Clerk {
 
       try {
         const sessionToken = cookies.get('__session');
-
         Logger.debug(`sessionToken: ${sessionToken}`);
 
-        if (!sessionToken) {
-          throw new HttpError(401, 'No session cookie found', undefined)
-        }
-
-        let sessionId: any = req.query._clerk_session_id;
+        const headerToken = req.get('Authorization')
+        Logger.debug(`headerToken: ${headerToken}`);
 
-        Logger.debug(`sessionId from query: ${sessionId}`);
-
-        let session: (Session | undefined) = undefined;
-        if (!sessionId) {
-          const client = await this.clients.verifyClient(sessionToken);
-          session = client.sessions.find(session => session.id === client.lastActiveSessionId);
-          Logger.debug(`active session from client ${client.id}: ${session?.id}`);
+        if (isAuthV2Request(this, sessionToken) || isAuthV2Request(this, headerToken)) {
```

Comment:
🔧 Not that it should make any difference, but consider switching the order for consistency with the Ruby SDK.